tests/integration/pkg/hooks/api-gateway.go

```diff
@@ -114,7 +137,7 @@ var ApplyAndVerifyApiGatewayCrSuiteHook = func() error {
 		}
 
 		return nil
-	}, testcontext.GetRetryOpts()...)
+	}, retry.DelayType(retry.FixedDelay), retry.Delay(500), retry.Attempts(1200))
```

Comment:
This seems like a very long time to retry the scale down. Why did we chose to make it that long?

Reply:
I will revert that to the old value